pkgs/by-name/fm/fm-go/package.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, stdenv
+}:
+
+let
+  finalAttrs = {
+    pname = "fm";
```

Comment:
would this pname not clash with the existing package? how might one specify which one to install?

Reply:
No, because there is no way 2 hashes of different packages would be the same.

Reply:
let me see. when installing a package i imagine one might execute e.g. `nix-env -iA fm`. as that type of operation does not involve the user specifying package hashes, how would nix know whether to install fm gtk or fm go?

Reply:
[Stop using nix-env.](https://stop-using-nix-env.privatevoid.net/)

For the sake of you and others.

Reply:
> let me see. when installing a package i imagine one might execute e.g. `nix-env -iA fm`. as that type of operation does not involve the user specifying package hashes, how would nix know whether to install fm gtk or fm go?

You should always use the derivation names instead of pnames to install, e.g.: `fm-go` instead. If `nix-env -iA` looks at the pnames instead, this is one more reason why it is a terrible mechanism to install packages.